scripts/us_cdc/heat_related_illness/README.md

```diff
@@ -0,0 +1,31 @@
+# Importing EPH Heat and Heat-related Illness data
+
+This directory imports [Heat and Heat-related Illness](https://ephtracking.cdc.gov/qrlist/35) from EPH Tracking into Data Commons. It includes data at a state level.
+
+## Cleaning source data
+To clean the source data, run:
```

Comment:
Pls indicate if the data needs to be downloaded manually when new files are available.

Reply:
Done.